x-pack/plugins/observability_solution/inventory/public/hooks/use_inventory_router.ts

```diff
@@ -47,7 +47,7 @@ export function useInventoryRouter(): StatefulInventoryRouter {
         navigateToApp('inventory', { path: next, replace: true });
       },
       link: (path, ...args) => {
-        return http.basePath.prepend('/app/observability/inventory' + link(path, ...args));
+        return http.basePath.prepend('/app/inventory' + link(path, ...args));
```

Comment:
IMO this entire hook could be replaced by the `entity_inventory_locator`. This hook is still prone to errors, should anything in the path changes

Reply:
the hook exists for typed navigation, do you mean that you are reproducing the types in `entity_inventory_locator`?

Reply:
>do you mean that you are reproducing the types in entity_inventory_locator

No, we are not. 

I think the hook still makes sense.

Reply:
nvm. I thought this was being used in the broken link.